### ISSUE TITLE
Narrator is not announcing the group name while navigating in different list items in drop down of ComboBox

### ISSUE THREAD

**Issue by vishakha116** [microsoft/fluentui#19062]
### Describe the issue:

On expanding the drop down in the combobox and going over the items in the dropdown, the group header is not announced as a separate group name. It is simply announced as some other item in the list. People with visual impairment will not be able to know that different groups are available in the combobox dropdown list while navigating using up/down arrow key. 

### Please provide a reproduction of the issue in a codepen:

[https://codepen.io/vishakhagupta/pen/bGWoooZ](https://codepen.io/vishakhagupta/pen/bGWoooZ)

#### Actual behavior:

On using up/down arrow key to navigate in the combobox drop down list narrator is not announcing the group name.

#### Expected behavior:

Narrator should announce the group name while navigating to different list items in the drop down list of the combobox.

**Comment by ling1726** [comment]
This might be by design. In scan mode Narrator can read all the text inside the combobox if necessary. Announcing group name with every single option in the menu would be a very verbose experience for Narrator users.

@smhigley would be nice to get your expertise here .🙏

**Comment by petewu100** [comment]
The ask is not to announce the group name with every single option. The ask is to provide proper hierarchy in the accessibility tree that will allow the screen reader to announce "context" information. For example, Narrator's default setting for context is "2 - Immediate Context". It will announce the name of the parent element when you navigate to an element under a different parent. So in this case the group name would only be announced when you navigate across groups and not when you navigate within a group.
